src/api.py

```diff
@@ -159,34 +210,28 @@ async def user_exists(mongo: Motor, username) -> bool:
 
 
 async def search_posts(
-    mongo: Motor, username: str, content: str, page: int, behavior: SearchBehavior
-) -> Tuple[int, list[Post], bool]:
+    mongo: Motor,
+    username: str,
+    content: str,
+    page: int,
+    behavior: SearchBehavior,
+    mentions: bool,
+) -> Tuple[int, list[Post]]:
     """
     Search for posts by username and/or content.
 
     :param mongo:
     :param username:
     :param content:
     :param page:
-    :param behavior: If SearchBehavior.USERNAME_AGGRESSIVE, content will be ignored.
+    :param behavior: If SearchBehavior.MATCH_ANY, return content matching any
+                     search field, if SearchBehavior.MATCH_ALL, return content
+                     matching all search fields.
```

Comment:
Nit: mentions is missing from the params list here (maintaining these manually is such a pain)